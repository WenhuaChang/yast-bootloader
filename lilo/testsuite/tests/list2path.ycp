/*
 * Authors:
 *  Unknown <unknown@addres.xx>
 *
 * $Id$
 *
 */

{
    include "testsuite.ycp";
    include "lilo/routines.ycp";

    TEST(``(list2path(["a","b","c"])), [], nil);
    TEST(``(list2path([])), [], nil);
    TEST(``(list2path(["a b", "c d", "e"])), [], nil);
    TEST(``(list2path(["a", "1", "b"])), [], nil);
    TEST(``(list2path(["a.b", "c", "d.e"])), [], nil);
}
