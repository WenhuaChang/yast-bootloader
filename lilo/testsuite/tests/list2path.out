Return	."a"."b"."c"
Return	.
Return	."a b"."c d"."e"
Return	."a"."1"."b"
Return	."a.b"."c"."d.e"
